index.js

```diff
@@ -14,14 +16,36 @@ function sortByStringLength() {}
  * Example: "Hello"
  * [ 'elloH', 'lloHe', 'loHel', 'oHell', 'Hello' ]
  */
-function textScroller() {}
+  function textScroller(word) {
+    let arrOfShuff = [];
+    if(word.length < 1){
+      return arrOfShuff;
+    }
+  for(let i = 1; i < word.length; i++) {
+    const shuffledWord = word.slice(i)+ word.slice(0,i);
```

Comment:
See if you can do this by splitting the string into an array.

While your solution is techincally faster - splitting the string into an array and removing the first element is a _very_ good habit to get used to when we get to DSA.